pom.xml

```diff
@@ -10,18 +10,20 @@
 	</parent>
 	<groupId>io.jenkins.plugins</groupId>
 	<artifactId>inodes-monitor</artifactId>
-	<version>0.6-SNAPSHOT</version>
+	<version>${revision}</version>
 	<packaging>hpi</packaging>
 
 	<name>Inodes Monitor Plugin</name>
 	<description>
 	This plugin adds a column for inodes usage monitoring for Linux/Unix systems. Like for the existing disk space monitors,
 	it can help put automatically a node offline where not enough inodes are still available.
 	</description>
-	<url>https://wiki.jenkins.io/display/JENKINS/Inodes+Monitor+Plugin</url>
+	<url>https://plugins.jenkins.io/inodes-monitor/</url>
 
 	<properties>
-		<jenkins.version>2.361.4</jenkins.version>
+		<revision>0.5.2</revision>
```

Comment:
The revision is not needed when using the automatic release process.  It is computed based on the number of commits in the repository and the SHA1 of the release commit.
```suggestion
```